toffy/bin_extraction.py

```diff
@@ -19,8 +21,13 @@ def extract_missing_fovs(bin_file_dir, extraction_dir, panel, extract_intensitie
     fovs = io_utils.remove_file_extensions(io_utils.list_files(bin_file_dir, substrs='.bin'))
     extracted_fovs = io_utils.list_folders(extraction_dir, substrs='fov')
 
-    # filter out moly fovs
-    moly_fovs = list_moly_fovs(bin_file_dir)
+    # filter out empty json file fovs
+    empty_fovs = check_for_empty_files(bin_file_dir, return_json_names=True, warn=True)
+    if empty_fovs:
+        fovs = list(set(fovs).difference(empty_fovs))
+
+    # check for moly fovs
+    moly_fovs = list_moly_fovs(bin_file_dir, fovs)
```

Comment:
Add a print statement for empty_fovs that are skipped. Also, the print statements should be conditional on non-empty lists